src/components/Accordion/Accordion.Link.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import Section from './Accordion.Section'
+import Title from './Accordion.Title'
+
+// Higher-order wrapper to render an enhanced Accordion.Section and
+// Accordion.Title
+export const Link = props => {
+  const { children, isSeamless: isSeamlessProp, to, href, ...rest } = props
+  const isLink = !!(to || href)
+  const isSeamless = isLink ? false : isSeamlessProp
+  const isOpen = false
+
+  const sectionProps = {
+    ...rest,
```

Comment:
What is `...rest` ?

Reply:
@jmauerhan It's a way to [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) an Array or Object in JS.

```js
// Original object
const obj = {
  a: 1,
  b: 2,
  c: 3
}

// Plucking out something, and assigning new const (variables)
const { a, ...rest } = obj

// const a = 1
// const rest = {
//   b: 2,
//   c: 3
// }
```

Reply:
Thanks for that explanation :)